creator-node/src/routes/files.js

```diff
@@ -172,208 +167,106 @@ const getCID = async (req, res) => {
   const CID = req.params.CID
   const trackId = parseInt(req.query.trackId)
 
-  const decisionTree = [{ stage: `BEGIN`, time: `${Date.now()}` }]
-  const logPrefix = `[getCID] [CID=${CID}]`
+  const prometheusResult = { result: null, mode: 'default' }
 
   /**
    * Check if CID is servable from BlacklistManager; return error if not
    */
-  let startMs = Date.now()
   const BlacklistManager = req.app.get('blacklistManager')
   const isServable = await BlacklistManager.isServable(CID, trackId)
-  decisionTree.push({
-    stage: `BLACKLIST_MANAGER_CHECK_IS_SERVABLE`,
-    time: `${Date.now() - startMs}ms`
-  })
   if (!isServable) {
-    decisionTree.push({
-      stage: `CID_IS_BLACKLISTED`
-    })
-
-    logGetCIDDecisionTree(decisionTree, req)
-    return sendResponse(
+    prometheusResult.result = 'abort_delisted'
+    return sendResponseWithMetric(
       req,
       res,
-      errorResponseForbidden(
-        `${logPrefix} CID has been blacklisted by this node`
-      )
+      errorResponseForbidden('CID has been delisted by this node'),
+      prometheusResult
     )
   }
 
+  const storagePaths = []
+
   // Compute expected storagePath for CID
-  let storagePath
   try {
-    storagePath = await DiskManager.computeFilePath(CID, false)
-    decisionTree.push({
-      stage: `COMPUTE_FILE_PATH_COMPLETE`
+    const currentStoragePath = await DiskManager.computeFilePath(CID, false)
+    storagePaths.push({
+      path: currentStoragePath,
+      pathType: 'new_storage_path'
     })
-  } catch (e) {
-    decisionTree.push({
-      stage: `COMPUTE_FILE_PATH_FAILURE`
+
+    const legacyStoragePath = await DiskManager.computeLegacyFilePath(CID)
```

Comment:
promise.all these two file path computations? (can do as fast follow)

Reply:
i think it's okay to leave as is without the `Promise.all` because it's 2 explicit calls. this pattern of two explicit calls is also used currently on master. it also gets just a tad more annoying with updating the pathType too